changelog_page.php

```diff
@@ -232,8 +229,11 @@ function print_project_header_changelog ( $p_project_name ) {
 
 			# check limit_Reporter (Issue #4770)
 			# reporters can view just issues they reported
-			if( ON === $t_limit_reporters && $t_user_access_level_is_reporter &&
-							!bug_is_user_reporter( $t_row['id'], $t_user_id )) {
+			$t_limit_reporters = config_get( 'limit_reporters' );
```

Comment:
yes... i moved those lines closer to the action... and missed to keep them outside of the loop